Week-1/Session-02-HTML_Deployment/Assignments/Intermediate/poem/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Poem</title>
+</head>
+<body>
+    <section>
+        <p>
```

Comment:
Here instead of the single <p> tag you and write <br> you can leverage the block scope elements. 

Like, you can create div and inside div you can create P tag for each sentence. So you can avoid the br tag.